i18n/de/docusaurus-plugin-content-docs/current/toolbox/accessibility_indicators/connectivity.md

```diff
@@ -94,42 +94,42 @@ For further insights into the Routing algorithm, visit [Routing/Bicycle](../../r
 
 #### Pedelec
 
-Considers all paths accessible by pedelec. This routing mode takes into account the surface and smoothness of streets while computing accessibility. For heatmaps, a pedelec speed of 23 km/h is assumed.
+Berücksichtigt alle mit dem Pedelec befahrbaren Wege. Dieser Routing-Modus berücksichtigt bei der Berechnung der Erreichbarkeit die Oberfläche und Glätte der Straßen. Für Heatmaps wird eine Pedelec-Geschwindigkeit von 23 km/h angenommen.
 
-:::tip Hint
+:::tip Hinweis
 
-For further insights into the Routing algorithm, visit [Routing/Bicycle](../../routing/bicycle). In addition, you can check this [Publication](https://doi.org/10.1016/j.jtrangeo.2021.103080).
+Weitere Einblicke in den Routing-Algorithmus erhalten Sie unter [Routing/Fahrrad](../../routing/fahrrad). Darüber hinaus können Sie diese [Publikation](https://doi.org/10.1016/j.jtrangeo.2021.103080) lesen.
 
 :::
 
 </TabItem>
 
 <TabItem value="car" label="Car" className="tabItemBox">
 
-#### Car
+#### Auto
 
-Considers all paths accessible by car. This routing mode takes into account speed limits and one-way access restrictions while computing accessibility.
+Berücksichtigt alle mit dem Auto befahrbaren Wege. Dieser Routing-Modus berücksichtigt bei der Berechnung der Erreichbarkeit Geschwindigkeitsbegrenzungen und Einbahnstraßenbeschränkungen.
 
-:::tip Hint
+:::tip Hinweis
 
-For further insights into the Routing algorithm, visit [Routing/Car](../../routing/car).
+Weitere Einblicke in den Routing-Algorithmus erhalten Sie unter [Routing/Car](../../routing/car).
 
 :::
 
 </TabItem>
 
 </Tabs>
 
-### Configuration
+### Konfiguration
 
 <div class="step">
   <div class="step-number">4</div>
   <div class="content">Choose a <code>Travel Time Limit</code> for your heatmap. This will be used in the context of your previously selected <i>Routing Type</i>.</div>
 </div>
 
-:::tip Hint
+:::tip Hinweis
```

Comment:
@LucasLomnicki I see that you've used "Tipp" in some places and "Hinweis" in others. Do let me know if there's a reason to switch between the two in German, but since they both replace the English word "Hint", maybe we should use only one term for consistency.